doc/releases/mimic.rst

```diff
@@ -69,6 +69,35 @@ Notes
 Instructions
 ~~~~~~~~~~~~
 
+#. If your cluster was originally installed with a version prior to
+   Luminous, ensure that it has completed at least one full scrub of
+   all PGs while running Luminous.  Failure to do so will cause your
+   monitor daemons to refuse to start.
```

Comment:
Could you clarify in which situation the MONs will refuse to start? After upgrading to Mimic? I'd also suggest to add a short explanation about *why* they will actually fail.

Reply:
Isn't that sufficiently explained two paragraphs down? If the `recovery_deletes` and `purged_snapdirs` flags aren't both present in the OSD map, then the mons don't start. (OK, they *do* start but they refuse to join the quorum, but for all intents and purposes that's functionally equivalent.)

Reply:
I blame it on my lack of in-depth Ceph expertise, but it wasn't clear to me that running a scrub will add the missing flags. It may help to make that connection in the explanation.